services/explorer/src/pages/Nodes/Nodes.tsx

```diff
@@ -81,7 +81,7 @@ const NodesPage = (): JSX.Element => {
         />
         <TablePagination
           rowsPerPageOptions={[25, 50, 75, 100]}
-          component='div'
+          component={'div' as any}
```

Comment:
Can we maybe get rid of this any somehow?

Reply:
For the moment no. In the documentation pasing `div` is corect but here it complains so this was the only fix. This should be resolved once we upgrade MUI and maybe resolve all dependencies issues